src/neofs_testlib/env/env.py

```diff
@@ -612,6 +619,7 @@ def start(self):
         self._generate_config()
         logger.info(f"Launching HTTP GW: {self}")
         self._launch_process()
+        logger.info(f"Launched HTTP GW: {self}")
```

Comment:
@evgeniiz321 Could you please take a look at these changes? I'm not sure if you'll be happy with them, but to check the logs, I need to know the GW properties after _launch_process (for example, self.stderr is empty before). Also, any feedback is more than welcome, as this is my first attempt at doing something in Python.

Reply:
This is fine

Reply:
doesn’t directly relate to adding REST, maybe a separate commit?